content/chainguard/chainguard-images/security-advisories/index.md

```diff
@@ -0,0 +1,161 @@
+
+```
+. . .
+
+20 vulnerabilities found in 6 packages
+  UNSPECIFIED  8  
+  LOW      	1  
+  MEDIUM   	3  
+  HIGH     	6  
+  CRITICAL 	2  
+```
+
+This output shows that this particular image has 20 vulnerabilities. The Docker Scout output also lists each of the packages affected by CVEs as well as the specific vulnerabilities it found for each.
+
+Scrolling to the listing immediately before the final vulnerability count, we'll find that in the case of this example, the package `nghttp2` is referenced. 
+
+```
+   0C 	1H 	0M 	0L 	1?  nghttp2 1.56.0-r0
+pkg:apk/chainguard/nghttp2@1.56.0-r0?os_name=chainguard&os_version=rolling
+
+	✗ HIGH CVE-2023-44487
+  	https://scout.docker.com/v/CVE-2023-44487
+  	Affected range : <1.57.0-r0  
+  	Fixed version  : 1.57.0-r0   
+    
+	✗ UNSPECIFIED GHSA-qppj-fm5r-hxr3
+  	https://scout.docker.com/v/GHSA-qppj-fm5r-hxr3
+  	Affected range : <1.57.0-r0  
+  	Fixed version  : 1.57.0-r0   
+    
+```
+
+As of this writing, this lists one vulnerability associated with version `1.56.0` of `nghttp2` as being `HIGH` severity and one as `UNSPECIFIED`. We'll use the `HIGH` severity vulnerability listed here as an example when we explore Chainguard's Security Advisories in the next section.
+
+Copy or note down the CVE identifier (`2023-44487` in this case). Additionally, note down the name of the affected package (`nghttp2`). You'll use these details to retrieve more information about the CVE shortly. 
+
+
+## Searching the Security Advisories
+
+After finding a vulnerability in a Chainguard Image, you can navigate to [Chainguard's **Security Advisories** page](https://images.chainguard.dev/security). This is a helpful resource you can use to determine the status for any CVE found within a Chainguard Image.
+
+The Security Advisories page is self-service, allowing you to check whether Chainguard is aware of a specific vulnerability and whether it has been mitigated in a certain package version. You can search the Security Advisories page by entering any CVE identifier to find what packages are affected by that CVE. You can also enter the names of individual packages to find what CVEs have been reported within them.
+
+Enter the CVE identifier you copied previously into the search box at the top of the page. This will immediately filter the list of security advisories to only show packages where that CVE has been reported. It will also show the **Status** of each.
+
+![Screenshot showing the Chainguard Security Advisories page. The list of advisories has been filtered to only include those with a CVE ID of "CVE-2023-44487".](sec-advisories-1.png)
+
+If you click on any row in the filtered list, it will take you to the CVE's specific page. There, you'll find a list of every package where this CVE has been reported.
```

Comment:
(offtopic) This is correct, but it's a little confusing, so I wonder if it will change in the future...

Reply:
Yeah I was struggling with describing things in this section. The joys of documenting UI flows. Definitely something to keep tabs on.